src/Stratis.Bitcoin.Cli/Program.cs

```diff
@@ -8,72 +9,144 @@
 using Newtonsoft.Json;
 using Stratis.Bitcoin.Configuration;
 using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Utilities.Extensions;
 
 namespace Stratis.Bitcoin.Cli
 {
     class Program
     {
         /// <summary>
         /// The expected sequence of arguments:
-        ///    1) [network-name] [rpc-command] [rpc-params].
+        ///    1) [network-name] [options] [rpc-command] [rpc-params].
```

Comment:
Hi @andrasfuchs! Ok so here rather use ```<list>``` and ```<item>```:

```
/// <summary>
/// The expected sequence of arguments:
/// <list>
/// <item> 1) [network-name] [options] [rpc-command] [rpc-params].

````

Reply:
Hi @fassadlr , thank you for taking the time for the code review!
Sure, I didn't know about this summary tag.